.ci/kata_chronic.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Use chronic to mute the output of a command, unless it errors.
+# But, to prevent system timeouts from complete inactivity, periodically
+# output '.'s to keep the system looking alive...
+
+cmdLine=$@
+
+chronic ${cmdLine[@]} &
```

Comment:
@grahamwhaley - I think the fix you need is to use `eval`:

```
eval chronic ${cmdLine[@]} &
```

Reply:
Thanks @jodh-intel - that seems to have done the trick. I re-massaged all the make lines with `&&` or `||` requirements into `bash -c` lines to get rid of the `@echo` faux-pas as well.
I didn't do the `printf ".\b"` update though - as I think I'd have to invest time to make it 'nice' in the log output and keep some semblance of printing 'how long did it actually take'. We can PR updates to that later if we find we need.
/test